docs/configuration.asciidoc

```diff
@@ -1054,13 +1069,33 @@ The default unit for this option is `ms`
 # This is used to keep all the errors and transactions of your service together
 # and is the primary filter in the Elastic APM user interface.
 # 
-# NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+$. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces.
+# The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]+$`. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces.
+# 
+# NOTE: When relying on auto-discovery of the service name in Servlet environments,
+# there is currently a caveat related to metrics.
+#  +
+# The consequence is that the 'Metrics' tab of a service does not show process-global metrics like CPU utilizationif the corresponding service name has been auto-discovered based on the `web.xml`'s `display-name` or the servlet context path.
+# The reason is that metrics are reported with the detected default service name,
+# for example `tomcat-application`.
+#  +
```

Comment:
Maybe add a clarification that this means system metrics can still be viewed when service names are detected automatically, but through the fallback service name.